admin/sql/CreateTables.sql

```diff
@@ -382,6 +382,11 @@ CREATE TABLE edit_note
     post_time            TIMESTAMP WITH TIME ZONE DEFAULT NOW()
 );
 
+CREATE TABLE edit_note_recipient (
+    editor_id           INTEGER NOT NULL, -- PK, references editor.id
```

Comment:
I realized we don't really use this `_id` naming scheme in other tables, we'd just use `editor` and `edit_note` and know it means the primary key, so maybe we should rename them for consistency.